### PR TITLE
Add "Setup a Firewall with IPFW on FreeBSD 12"

### PR DESCRIPTION
**Pull request by youiopmop**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: youiopmop rosethorn@riseup.net

### CODE REVIEW

File: tutorials/setup-a-firewall-with-ipfw-on-freebsd-12/01.en.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+```
+firewall_enable="YES"
+firewall_quiet="YES"
+firewall_type="workstation"
+firewall_allowservices="any"
+firewall_myservices="22/tcp"
+firewall_logdeny="YES"
+```
+This will setup a firewall using `workstation` as the type of firewall, and denying access to all ports from outside hosts except for the default ssh port; port 22.
```
Comment (n-se) [inline tutorials/setup-a-firewall-with-ipfw-on-freebsd-12/01.en.md]: Does the type workstation include anything else or is this just the name of the firewall?

File: tutorials/setup-a-firewall-with-ipfw-on-freebsd-12/01.en.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+*If you have changed the default SSH port from 22 to something else, edit the above to reflect that change or you may lock yourself out of your server when you start the firewall. In case you did this, just reboot into a rescue system or LiveCD and edit `/etc/rc.conf`.*
+
+Make sure to add any other ports or services (from `/etc/services`) that you need to access into `firewall_myservices`.
```
Comment (n-se) [inline tutorials/setup-a-firewall-with-ipfw-on-freebsd-12/01.en.md]: How would I do that?

Just another firewall_myservices="80/tcp" or does it has to be in one line? Can you make an example?

Comment (youiopmop) [reply tutorials/setup-a-firewall-with-ipfw-on-freebsd-12/01.en.md]: All on the same line, and space separated.
For example 
`firewall_myservices="80/tcp 22/tcp 1111/udp 9999"` for 22 (ssh) on TCP, 80 (http) TCP, 1111 UDP, and 9999 TCP and UDP. You can also use the name of services from `/etc/services` e.g. `firewall_myservices="http ssh 1111/udp 9999"`

File: tutorials/setup-a-firewall-with-ipfw-on-freebsd-12/01.en.md (hunk 1 above)
Comment (youiopmop) [reply tutorials/setup-a-firewall-with-ipfw-on-freebsd-12/01.en.md]: The different firewall types are explained on the [IPFW documentation for FreeBSD](https://www.freebsd.org/doc/handbook/firewalls-ipfw.html).

`workstation: protects only this machine using stateful rules.`

in my experience for a server `workstation` is the best default configuration to use, without having to mess around with using a `firewall_script`, which may be difficult for people new to FreeBSD/ipfw. I included a link in the conclusion to the FreeBSD documentation of IPFW if people want to configure ipfw further then what is described in the article.

**Pull request merged**